src/routes/(inner)/examples/popovers/Example.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+	import { fade } from 'svelte/transition';
+	import {
+		autoUpdate,
+		offset,
+		flip,
+		arrow,
+		useFloating,
+		FloatingArrow,
+		useInteractions,
+		useRole,
+		useDismiss,
+	} from '$lib/index.js';
+
+	// State
+	let open = $state(true);
+	let elemArrow: HTMLElement | null = $state(null);
+
+	// Use Floating
+	const floating = useFloating({
+		whileElementsMounted: autoUpdate,
+		get open() {
+			return open;
+		},
+		onOpenChange: (v) => (open = v),
+		placement: 'top',
+		get middleware() {
+			return [offset(10), flip(), elemArrow && arrow({ element: elemArrow })];
+		},
+	});
+
+	// Interactions
+	const dismiss = useDismiss(floating.context);
+	const role = useRole(floating.context, { role: 'menu' });
+	const interactions = useInteractions([role, dismiss]);
+</script>
+
+<div>
```

Comment:
Go ahead and add the spacing classes here like the Tooltip so things don't bump together.

Note: I'm down to write the text copy for the page later this week.

Reply:
I don't see any spacing classes in the Tooltip example. Am I missing something?
> Note: I'm down to write the text copy for the page later this week.

Sure!

Reply:
@Hugos68 sorry this was intended for the `/examples/popovers/+page.svelte`. My bad!